lib/framework/view/bottom_nav_page_group.dart

```diff
@@ -365,20 +368,21 @@ class EnsembleBottomAppBarState extends State<EnsembleBottomAppBar> {
 
     return Theme(
       data: ThemeData(useMaterial3: false),
-      child: ClipRRect(
-        borderRadius: widget.borderRadius ?? BorderRadius.zero,
+      child: Container(
+        margin: Utils.optionalInsets(widget.margin) ?? EdgeInsets.zero,
+        decoration: BoxDecoration(
+          borderRadius: widget.borderRadius ?? BorderRadius.zero,
+        ),
+        clipBehavior: Clip.hardEdge,
```

Comment:
use Clip.none if border radius is not specified. Minimizing clip if possible if we dn't need it.